### PR TITLE
Setting Up Kali Linux for dual boot

### LINKED ISSUE #2780: How to set up Kali Linux for dual boot
**Introduction**

Kali Linux is a very powerful, open-source penetration testing tool used by hackers and security specialist in day to day encounters. Before learning or accessing various tools in Kali Linux, you will need  to install Kali Linux in your pc.
In this tutorial, we will learn how to set up Kali Linux for dual boot in your pc. We will first learn how to set up a pen drive to be used in the  boot process 
We will then set up the dual boot for Kali Linux  and enter the appropriate options.

**Key Takeaways**

By the end of the tutorial, the user should be able to :

1.  Set up a pen drive to be used in a dual boot
2.  Successfully set up any  OS for a dual boot

**Reference**

N/A

### PR DESCRIPTION
**Pull request by Wkd-Cyber**
### Article checklist

**ATTENTION: In your PR - add the ISSUE (topic form) # that this PR corresponds to. You can do this by also adding a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.**
      
      
**NOTE: (Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**If you have not already please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve our overall technical writing.**

#### Preliminary Checks - Formatting and Structure 
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**
- [x] Have you included a hero image that is 600x400 pixels and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
- [x] In your PR - add a comment that states **["This fixes #(enter your Idea Content Form #)"](https://youtu.be/TKJ4RdhyB5Y?t=278)** to link and close your Idea Content (issue) when the PR is merged.
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) -  [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker? We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [ ] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**

### Contribution guidelines
For first-time contributors and for more details, **see [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
This fixed #2780

### CODE REVIEW

**Comment by Ericgacoki** [comment]
Greetings @Wkd-Cyber,
Welcome to the Eng-Ed program. We're excited to have you onboard. Thank you for submitting your `first` article for review.
Seems like you forgot to add author details 🤭. Please create your author folder in the `/content/authors` directory. This folder holds your author bio and an avatar. Feel free to check how other students have structured theirs.

Ping me when you're done.

**Comment by Wkd-Cyber** [comment]
Changes done @Ericgacoki 
Please continue with the review

**Comment by Ericgacoki** [comment]
@Wkd-Cyber, I've made a few changes to improve the flow of your article. Be sure to [take a look](https://github.com/section-engineering-education/engineering-education/pull/3113/commits/edac4624645f579685d3bad885314006840157d1).

File: content/articles/setting-up-kali-linux-for-dual-boot/index.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+3. Windows OS running on your PC or laptop.
+
+### Getting started
+Now, download the files required during the process.
+
+#### Downloading Kali Linux Iso image
+Navigate to the [Kali Linux official site](https://kali.org/get-kali/#kali-bare-metal) and download the latest version of Kali Linux.
+
+#### Creating a bootable pen drive
+We will first download software used to make a pen drive bootable. We'll use `Rufus` in this context. You can download [Rufus here](http://rufus.ie/en_US/).
```
Comment (Ericgacoki) [inline content/articles/setting-up-kali-linux-for-dual-boot/index.md:20]: Could you please explain why you chose `Rufus` over other software?

File: content/articles/setting-up-kali-linux-for-dual-boot/index.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+Wait for the installation set-up to load all components. This should take a short while.
+
+In the following setup, we'll use the recommended choices for an assured successful set-up.
+
+- Load missing firmware from removable media? - `No`.
+- Connect to your WiFi card if one is available - This is optional.
+
+Wait for the set-up to `configure all the Networks`. This process is also quick.
+
```
Comment (Ericgacoki) [inline content/articles/setting-up-kali-linux-for-dual-boot/index.md:60]: This only happens when the user decides to connect to a WI-FI network. Please clarify that.

File: content/articles/setting-up-kali-linux-for-dual-boot/index.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+Moving on, create a primary user for Kali Linux.
+
+- Set the `Host name` and `Domain name` as you wish - This is an optional step.
```
Comment (Ericgacoki) [inline content/articles/setting-up-kali-linux-for-dual-boot/index.md:63]: Explain to the readers what these terminologies mean.

File: content/articles/setting-up-kali-linux-for-dual-boot/index.md [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+- Select the `swap area` option as the use of the partition.
+- Remember to select `done setting up the partition`.
+
+At this point, you have successfully created a partition.
+Click `Finish partitioning and write changes to disk` then hit the continue button.
+
+On the **write changes to disk** option, select `Yes` and wait for the installer set-up to `install the base system`.
+
+### Software selection
+Here you have to choose your favorite desktop environment. In my case, I'll' choose the `Xfce (Kali default desktop environment)`.
```
Comment (Ericgacoki) [inline content/articles/setting-up-kali-linux-for-dual-boot/index.md:101]: Remember the reader just installed Kali Linux. They don't have a favorite desktop environment yet. Consider mentioning a few (if not all) environments and their differences.
Try to answer the question - Why this and not the other one?

File: content/articles/setting-up-kali-linux-for-dual-boot/index.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+### What happens after rebooting?
+After rebooting, Kali Linux will load into `GRUB boot loader`. This is where you're given the option to either boot into Kali Linux or Windows. Select Kali Linux in this case.
+
+The login screen will be displayed as shown below:
+![Login page](/engineering-education/setting-up-kali-linux-for-dual-boot/login.png)
+
+Enter your login credentials, that is, your `username` and `password` to proceed.
+
+Upon successful login, you will be able to access Kali Linux home screen on your machine. If it is your first time using Kali Linux, you can go through [this article](https://www.section.io/engineering-education/getting-started-with-kali-linux/) for more information.
+
```
Comment (Ericgacoki) [inline content/articles/setting-up-kali-linux-for-dual-boot/index.md:116]: Just before the summary, give general importance of installing a Linux-based OS alongside Windows.

**Review by Ericgacoki — CHANGES_REQUESTED** [review]
Hello @Wkd-Cyber, I've noted a few points that need more clarification to clear any reader's doubts. Generally, try to explain every bit no matter how minor it might seem to be.
Check out the comments attached below.

**Comment by Wkd-Cyber** [comment]
Changes made @Ericgacoki 
Please continue with the review

**Review by Ericgacoki — APPROVED** [review]

**Comment by Ericgacoki** [comment]
Greetings @Wkd-Cyber, this article will be ready for final review once `plagiarism check` is done.

cc: @hectorkambow.

**Comment by Ericgacoki** [comment]
@Wkd-Cyber, I've seen the changes ✅

**Comment by hectorkambow** [comment]
plagiarism check done

**Review by kayere — APPROVED** [review]
Congratulations for publishing your first article @Wkd-Cyber. The article is very easy to follow through. Keep up the good work.

Thank you for the review @Ericgacoki.

**Pull request merged**